src/content/products/sparseml.mdx

```diff
@@ -45,10 +45,19 @@ index: 2000
 
 ## Overview
 
-SparseML is a toolkit that includes APIs, CLIs, scripts and libraries that apply state-of-the-art [sparsification](https://docs.neuralmagic.com/main/source/getstarted.html#sparsification) algorithms such as pruning and quantization to any neural network.
-General, recipe-driven approaches built around these algorithms enable the simplification of creating faster and smaller models for the ML performance community at large.
+SparseML is a toolkit that includes APIs, CLIs, scripts and libraries that enable you to create sparse models trained on your data.
 
-The [GitHub repository](https://github.com/neuralmagic/sparseml) contains integrations within the PyTorch, Keras, and TensorFlow V1 ecosystems, allowing for seamless model sparsification.
+SparseML provides two avenues to accomplish this goal: 
+- **Sparse Transfer Learning**: Fine-tune state-of-the-art pre-sparsified models from the SparseZoo onto your dataset while preserving sparsity.
+
+- **Sparsifying from Scratch**: Apply state-of-the-art [sparsification](https://docs.neuralmagic.com/main/source/getstarted.html#sparsification) algorithms such as pruning and quantization to any neural network.
+
+These options are useful for different situations:
+- **Sparse Transfer Learning** is the easiest path to creating a sparse model trained on your data. Pull down a sparse model from SparseZoo and point our training scripts at your data without any hyperparameter search. This is the recommended pathway for supported use cases like Image Classification, Object Detection, and several NLP tasks. 
```

Comment:
@rsnm2 In certain places, these use cases are cited as proper nouns e.g., "Image Classification" but then elsewhere, they are simple modifiers or general references e.g., "image classification models" - may you do a find and replace to determine what is what here. I wouldn't have expected it to be proper nouns here for example?